lib/routes.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:go_router/go_router.dart';
+import 'package:picple/presentation/home/view/home_page.dart';
+import 'package:picple/presentation/login/view/login_page.dart';
+import 'package:picple/presentation/splash/view/splash_page.dart';
+
+final router = GoRouter(
+  routes: [
+    GoRoute(path: '/', builder: (context, state) => const SplashPage()),
+    GoRoute(path: '/home', builder: (context, state) => const HomePage()),
+    GoRoute(path: '/login', builder: (context, state) => const LoginPage()),
+  ],
+);
```

Comment:
C:
```suggestion
enum Routes {
  splash(name: 'Splash', path: '/'),
  home(name: 'Home', path: '/home'),
  login(name: 'Login', path: '/login');

  final String name;
  final String path;

  const Routes({required this.name, required this.path});
}

final router = GoRouter(
  routes: [
    GoRoute(path: Routes.splash.path, builder: (context, state) => const SplashPage()),
    GoRoute(path: Routes.home.path, builder: (context, state) => const HomePage()),
    GoRoute(path: Routes.login.path, builder: (context, state) => const LoginPage()),
  ],
);
```
매직넘버 대신 변수로 관리하는 것처럼 경로도 enum 으로 관리하면 좋을 것 같아
String으로 해서 사람이 직접 입력 시 휴먼 에러도 발생하기도 하고
아예 Routes에 정의되지 않은 경로를 사용 시에는 컴파일 타임에 에러를 잡아내니까

[추후 경로에 argument 넣을 때도 Args 클래스 만들어서 하면](https://mohaned.me/posts/type-safe-routes/) 좋을 듯!